lib/beaker/cli.rb

```diff
@@ -71,12 +69,16 @@ def print_version_and_options
       @logger.info(@options.dump)
     end
 
+    # Create a network manager instance
+    def create_network_manager
```

Comment:
There's a few things here:

- I'm guessing we shouldn't have both `initialize` and `create` for `$VERB_HEREnetwork_manager`. Seems confusing...maybe we should just leave the `@network_manager` in as an `:attr_reader` and expect subcommands to call them directly?
- With what is outlined in [BKR-1090](https://tickets.puppetlabs.com/browse/BKR-1090) we could feasibly run `@cli.initialize_network_manager` and then simply just run cleanup after that, without having to store additional files to disk